deploy/ui/form.json

```diff
@@ -120,7 +141,6 @@
                 {
                   "label": "url",
                   "type": "text",
-                  "required": true,
```

Comment:
Why are these 3 set to not being required any longer?

Reply:
Restored